src/cozmo/util.py

```diff
@@ -134,6 +134,17 @@ def degrees(self):
         '''float: The angle in degrees.'''
         return self._radians / math.pi * 180
 
+    @property
+    def abs_value(self):
+        """:class:`cozmo.util.Angle`: The absolute value of the angle.
+        
+        If the Angle is positive then it just returns the Angle, otherwise it
+        returns -Angle.
+        """
+        if self.__lt__(radians(0)):
```

Comment:
suggest changing this to the following (which doesn't have to do quite as much work):
```python
if self._radians < 0:
    return radians(-self)
```